### PR TITLE
resolve error in merge

### PR DESCRIPTION
**Pull request by jinwoo1289**
I made a mistake entering a duplicate code. Sorry and I've corrected it.

### CODE REVIEW

**Comment by jinwoo1289** [comment]
Please merge as soon as possible.

File: Invaders-SDP.iml
Comment (logpacket) [inline Invaders-SDP.iml:1]: Unnecessary change.

File: .idea/misc.xml
Comment (logpacket) [inline .idea/misc.xml:1]: Unnecessary change.

**Review by logpacket — CHANGES_REQUESTED** [review]
Please remove unnecessary change.

**Comment by jinwoo1289** [comment]
I solved it.

**Review by idealtrue0 — APPROVED** [review]

**Review by logpacket — APPROVED** [review]
